names/module.py

```diff
@@ -0,0 +1,210 @@
+import discord
+from discord.ext import commands
+
+from pie import check, utils, i18n, logger
+from modules.base.admin.database import BaseAdminModule
+from modules.boards.karma.database import KarmaMember
+
+from .database import Price
+
+_ = i18n.Translator("modules/fun").translate
+bot_log = logger.Bot.logger()
+
+
+class Nicknames(commands.Cog):
```

Comment:
The module name should be the same as the name of its directory. Since you are using `names` everywhere, the class name should be `Names` as well.

Reply:
Changed to names